jobs/model_create.rb

```diff
@@ -0,0 +1,53 @@
+require 'resque'
+require 'resque-status'
+require 'sinatra/activerecord'
+require 'redis'
+require_relative './reload_cache.rb'
+require_relative '../models/videos'
+require_relative '../models/articles'
+require_relative '../models/feeds'
+require_relative '../models/resources'
+
+module Jobs
+    module Models
+        class Create
+            include Resque::Plugins::Status
+            
+            #@queue = :models
+            
+            def perform
+                # Find object in database.
+                begin
+                    @type = options['model_type'].split('::').reduce(Module, :const_get)
+                    at(1, 3)
+                    @type.create!(options['args'])
+                    at(2, 3)
+                rescue ActiveRecord::RecordNotFound
+                    # Invalid ID.
+                    failed(msg: 'Model not found')
```

Comment:
Now that the frontend feeds directly from the job status, need to send the translated string. (Locale code will need to be passed through GET parameter.)